### PR TITLE
GSC Overview - Show GSC members and candidates

### PR DESCRIPTION
**Pull request by sentilesdal**
Initial implementation to show gsc members and candidates.  working with a lot of events in prod, a little worried about performance but let's see how it goes first.

I tested this with the local testnet, all those addresses are hardhat signers 1-8.
I made sure to test with a gsc member who was kicked to ensure they weren't listed.

image

### CODE REVIEW

File: src/elf-council-addresses/index.ts [hunk 1]
```diff
@@ -6,22 +6,6 @@ import {
...
-  //     timeLock: "0x7a2088a1bFc9d81c55368AE168C2C02570cB814F",
-  //     lockingVault: "0x67d269191c92Caf3cD7723F116c85e6E9bf55933",
-  //     vestingVault: "0xE6E340D132b5f46d1e472DebcD681B2aBc16e57E",
-  //     optimisticRewardsVault: "0x0000000000000000000000000000000000000000",
-  //     airdrop: "0x9E545E3C0baAB3E08CdfD552C960A1050f373042",
-  //     optimisticGrants: "0x0000000000000000000000000000000000000000",
-  //     treasury: "0xa82fF9aFd8f496c3d6ac40E2a0F282E47488CFc9",
-  //     spender: "0x0000000000000000000000000000000000000000",
-  //   },
-  // };
```
Comment (DannyDelott) [inline src/elf-council-addresses/index.ts:24]: deadcode

File: src/elf-council-tokenlist/testnet.tokenlist.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
       "symbol": "",
       "extensions": {
         "coreVoting": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
-        "votingPowerBound": "100",
+        "votingPowerBound": "100000000000000000000",
```
Comment (DannyDelott) [inline src/elf-council-tokenlist/testnet.tokenlist.json:76]: seems suspect? 😄

**Review by DannyDelott — APPROVED** [review]
looks good, one request for `select`

File: src/elf-council-tokenlist/testnet.tokenlist.json (hunk 2 above)
Comment (sentilesdal) [reply src/elf-council-tokenlist/testnet.tokenlist.json:76]: no its right, should be 100 eth value, not 100 wei value.

File: src/ui/gsc/useGSCMembers.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    enabled: !!membersArray?.length,
+  });
+
+  if (!members) {
+    return [];
+  }
+
+  return members.map((address) => ({
+    address,
+  }));
```
[the remark this answers is not in the document]
Comment (sentilesdal) [reply src/ui/gsc/useGSCMembers.tsx:54]: o right good call!

File: src/elf-council-addresses/index.ts (hunk 1 above)
Comment (sentilesdal) [reply src/elf-council-addresses/index.ts:24]: already removed in later commit

**Pull request merged**